python/dgl/distributed/dist_graph.py

```diff
@@ -315,11 +309,7 @@ def __init__(self, ip_config, graph_name, gpb=None, conf_file=None):
         else:
             connect_to_server(ip_config=ip_config)
             self._client = KVClient(ip_config)
-            g = _get_graph_from_shared_mem(graph_name)
-            if g is not None:
-                self._g = as_heterograph(g)
-            else:
-                self._g = None
+            self._g = _get_graph_from_shared_mem(graph_name)
```

Comment:
Why is this still `_get_graph_from_shared_mem`?

Reply:
i don't just share graph index. i also share some node/edge data. please check the function.